providers/aws/services/iam/iam_root_hardware_mfa_enabled/iam_root_hardware_mfa_enabled.py

```diff
@@ -0,0 +1,30 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.iam.iam_service import iam_client
+
+
+class iam_root_hardware_mfa_enabled(Check):
+    def execute(self) -> Check_Report:
+        findings = []
+        virtual_mfa = False
+        report = Check_Report(self.metadata)
+        report.region = "us-east-1"
+        report.resource_id = "root"
+        report.resource_arn = f"arn:aws:iam::{iam_client.account}:root"
+
+        if iam_client.account_summary["SummaryMap"]["AccountMFAEnabled"] > 0:
```

Comment:
Why don't you need to check if this value exists like in other checks?

Reply:
Because the key AccountMFAEnabled  always exists (if 0 MFA is disabled)

Reply:
Good!